File: Basic/Check if A Number is Prime or Not/SolutionByKaran.java
import java.util.*;
import java.lang.*;
import java.io.*;


class abc
{	
    static void prime(int n)
    {
        int k=0;
        for(int i=1;i<=n;i++)
        {
            if(n%i==0)
            {
                k++;
            }
        }
        if(k==2)
        {
            System.out.println("Yes");
        }
        else
        {
            System.out.println("No");
        }
	}
	public static void main (String[] args) throws java.lang.Exception
	{	
      int a;
      Scanner sc = new Scanner(System.in);
      System.out.println("Enter the number :");
      a = sc.nextInt();
      prime(a);
	}
}

File: Basic/Product of Digits in A Number/SolutionByKaran.cpp
#include <iostream>
using namespace std;
long product(long n)
{
    if ( n == 0 ) {
        return 0;
    }
    long p=1;
    while(n>0)
    {
        if(n%10 != 0) {
            p=p*(n%10);
        }
        n=n/10;
    }
    return p;
}
int main()
{
    long a;
    cout<<"Enter the number: "<<endl;
    cin>>a;
    cout<<"The product is: "<<product(a) << endl;
    return 0;
}
